packages/create-cloudflare/e2e-tests/frameworks.test.ts

```diff
@@ -353,7 +364,10 @@ describe.concurrent(`E2E: Web frameworks`, () => {
 					}
 				}
 			},
-			{ retry: 1, timeout: timeout || TEST_TIMEOUT }
+			{
+				retry: process.env.E2E_NO_RETRY ? 0 : TEST_RETRIES,
```

Comment:
would it make sense to have `E2E_RETRIES` instead of `E2E_NO_RETRIES` (and parse is like `parseInt(process.env.E2E_RETRIES)`) and so make this more configurable? (and if you want no retries you can do `E2E_RETRIES=0`) 🤔 

overkill?

Reply:
Good idea